docs/developer-guide/syntax-reference.rst

```diff
@@ -1618,7 +1626,10 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | RANDOM                 |  ``RANDOM()``                                                             | Return a random DOUBLE value between 0.0 and 1.0. |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
-| ROUND                  |  ``ROUND(col1)``                                                          | Round a value to the nearest BIGINT value.        |
+| ROUND                  |  ``ROUND(col1)`` or                                                       | Round a value to the nearest BIGINT value.        |
+|                        |  ``ROUND(col1, precision)``                                               |                                                   |
+|                        |                                                                           | precision is a number indicating up to how many   |
```

Comment:
How about: "precision is an INT indicating the number of decimal places to round to."

(For consistency)

Reply:
Same precision name opinion as CEIL (see above)